src/main.tsx

```diff
@@ -26,6 +27,8 @@ ReactDOM.render(
       <ReactQueryDevtools initialIsOpen={false} />
       <Router>
         <Routes>
+          <Route path="/login" element={<Login />}/>
+          <Route path="/signUp" element={<Login />}/>
```

Comment:
Each path needs to be a separate component.
You cant use Login for /login and /signup

/signUp => /signup

Reply:
done